### PR TITLE
README: big clean-up and adding fish shell instructions

### PR DESCRIPTION
**Pull request by Anton-Latukha**
- [x] Archlinux -> Arch Linux
- [x] Unified the form of headings (no longer in different cases and different forms)
- [x] Unified form of code blocks
- [x] Removed a code block syntax highlighting from one block. GitHub shell code blocks syntax highlighting is messy and inconsistent. Removing highlighting from that code block also creates a unified look for all code blocks. 
- [x] Note on the screenshots that terminal output depends on font and terminal. In my terminal setup it looks pretty. In-repo screenshots have too pixelated `system`-like font, which is not best show-case. 'cmon, just look at `.sig` and `.signature` italics.
- [x] For screenshots, HTML replacement text changed. From not understandable 'LS_COLORS' to Screenshot'. 
- [x] Added 'Table of contents'
- [x] Instructions for `fish` shell users. `fish` is non-POSIX compliant shell, so needs special instructions.

### CODE REVIEW

**Comment by Anton-Latukha** [comment]
You can view result at my [branch](https://github.com/Anton-Latukha/LS_COLORS/tree/add_fish).

File: README.markdown [hunk 1]
```diff
@@ -1,8 +1,16 @@
-LS_COLORS
-=========
-
+### Table of contents
+* [LS_COLORS](#ls_colors)
+* [How does it look like](#how-does-it-look-like)
```
Comment (rpdelaney) [inline README.markdown]: The original sounds more natural to my ear. "What does it look like?" is more native in English than "How does it look like?"

File: README.markdown [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+  * [Arch Linux](#arch-linux)
+  * [ZSH syntax highlighting](#zsh-syntax-highlighting)
+  * [fish shell](#fish-shell)
+* [Information for Developers](#information-for-developers)
+* [Legal](#legal)
+
+# LS_COLORS
 This is a collection of extension:color mappings, suitable to use as your
-LS COLORS environment variable. Most of them use the extended color map,
+`LS COLORS` environment variable. Most of them use the extended color map,
```
Comment (rpdelaney) [inline README.markdown]: Please change this to `LS_COLORS`

File: README.markdown [hunk 3]
```diff
@@ -19,63 +27,64 @@ For files that usually ends up next to each other, like html, css and js,
 try to pick colors that fits in nicely together. Filetypes with multiple
 possible extensions, like htm and html, should have the same color.
 
+# How does it look like
```
Comment (rpdelaney) [inline README.markdown]: As above, this should be "What does it look like?"

File: README.markdown [hunk 4]
```diff
@@ -19,63 +27,64 @@ For files that usually ends up next to each other, like html, css and js,
...
+  [1]: https://github.com/trapd00r/File-LsColor
+  [2]: https://github.com/trapd00r/File-LsColor/tree/master/bin
+  [3]: https://aur.archlinux.org/packages/lscolors-git
+
 # Installation
 To install and use this file, put something like this is your shell resource
 file:
-
-    eval $( dircolors -b $HOME/LS_COLORS )
+```
```
Comment (rpdelaney) [inline README.markdown]: Perhaps this should be ` ```sh `?

File: README.markdown [hunk 5]
```diff
@@ -19,63 +27,64 @@ For files that usually ends up next to each other, like html, css and js,
...
+eval $( dircolors -b $HOME/LS_COLORS )
+```
 
 A quick way to install would be the following set of commands (assuming bash):
 
-```sh
+```
```
Comment (rpdelaney) [inline README.markdown]: Why this change?

File: README.markdown [hunk 6]
```diff
@@ -19,63 +27,64 @@ For files that usually ends up next to each other, like html, css and js,
...
-=======================
-
+## ZSH syntax highlighting
 [zsh-syntax-highlighting-filetypes][0] highlights file on the command-line in
 realtime, using these colors.
 
-DEVELOPERS
-==========
+## fish shell
+```
```
Comment (rpdelaney) [inline README.markdown]: ` ```sh `? I don't know much about the `fish` shell or if something else would be more appropriate for that, though.

File: README.markdown (hunk 4 above)
Comment (rpdelaney) [reply README.markdown]: Also applies to the other cases where ` ``` ` occurs.

Comment (Anton-Latukha) [reply README.markdown]: As for me, I am experienced in shell scripting, with particularly good knowledge of the POSIX scripting, but I can not get heads or tails with `Linguist` highlighting for `shell`. The highlighting is very inconsistent.

If I use `sh` for code blocks - I also need to enable highlighting for `fish` block. But the `sh` and `fish` highlighting are mutually inconsistent.

Comment (Anton-Latukha) [reply README.markdown]:
```sh
eval $( dircolors -b $HOME/LS_COLORS )
```

And

```fish
eval ( dircolors -c $HOME/LS_COLORS )
```

File: README.markdown (hunk 5 above)
Comment (Anton-Latukha) [reply README.markdown]: Explained above - for a unified look, for some consistency.

File: README.markdown (hunk 6 above)
Comment (Anton-Latukha) [reply README.markdown]: Protip: I always consult https://github.com/github/linguist/blob/master/lib/linguist/languages.yml page.

File: README.markdown (hunk 1 above)
Comment (Anton-Latukha) [reply README.markdown]: Ok. I thought "SO WHAT DOES IT LOOK LIKE" is too informal.
Probably I aimed at "How does it look?".
I agree with "What does it look like?".

File: README.markdown (hunk 2 above)
Comment (Anton-Latukha) [reply README.markdown]: True

File: README.markdown (hunk 1 above)
Comment (Anton-Latukha) [reply README.markdown]: Take a notice that `?` (& other symbols) are not in the HTML anchors.

File: README.markdown (hunk 4 above)
Comment (rpdelaney) [reply README.markdown]: I'm sympathetic to your concerns, as are many others. I don't like their scheme at all either and I frequently find myself doing silly things like `curl <github_url> | vim -` just so I can actually read a piece of code. But...

1. I think the scheme they use, as inadequate as it is, is better than nothing, and
2. It may be improved in the future, and
3. Other markdown viewers might use better syntax highlighting, but won't be able to if we don't include the type hints.

With all that said I'm afraid I have to insist on the type hints.

File: README.markdown (hunk 1 above)
Comment (Anton-Latukha) [reply README.markdown]: I've just made a note that symbols get dropped in anchors, that this is right code.

File: README.markdown (hunk 4 above)
Comment (Anton-Latukha) [reply README.markdown]: Done

**Comment by rpdelaney** [comment]
Thank you!
